docs/metrics/index.md

```diff
@@ -0,0 +1,150 @@
+will truly answer the question.  Often we'll come into an experiment with ideas
+on how to measure something and after writing this document realize that there
+are better options.
+
+The document is expected to be at the `docs/metrics.md` path from the root of
+the repository.  Ideally it includes actual sketches of dashboard visuals the
+experiment leads would like to see.  It should include any special
+circumstances or requirements in the experiment, as well as details on how the
+metrics will be collected (keys, schemas, names of elements, etc.)
+
+Several examples exist
+([1](https://github.com/meandavejustice/min-vid/blob/master/docs/metrics.md),
+[2](https://github.com/mozilla/universal-search/blob/master/docs/metrics.md),
+[3](https://github.com/internetarchive/FirefoxNoMore404s/blob/master/docs/metrics.md))
+but it's important to understand that this is an evolving process and these
+examples may not represent the current needs (for example, none of the above
+leverage Google Analytics).
+
+The Metrics plan should be looked at by the Mozilla Measurement & Analytics
+Teams to make sure avoid implementation troubles.
+
+# Implementing Metrics in an Experiment
+
+The Metrics Plan you've written should serve as a blueprint for implementation.
+
+## Telemetry
+To use Telemetry you'll need to make sure the Mozilla Measurement Team is aware
+a new experiment is starting up.  That involves filing a bug to get Test Pilot
+Experiment data into re:dash, and then creating a re:dash dashboard.  Feel free
+to [clone these two bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=1297199%2C+1297200).
+
+The Measurement Team will be looking at the Metrics Plan you wrote to
+understand what is needed so it's important that it includes the schema you'll
+be logging your data to.
+
+## Google Analytics
+To use Google Analytics you'll need to have your Google Analytics credentials
+to record data.  Using your own account here instead of a generic Test Pilot
+account makes the transition easy when your experiment graduates from the Test
+Pilot program.  (Note that for the extent of the Test Pilot experiment you'll
+need to grant access to analytics for the Test Pilot team).
+
+In order to take full advantage of Google Analytics it's important to follow
+the Metrics Plan you outlined above -- this will let you use the built-in
+e-commerce, funnel, and goal conversion features which will lead to speedy
+analysis.
+
+## Sending Data
+
+The actual function to report event based data is called `testpilot.sendEvent()` and is
+provided by the `test-pilot-metrics` node module. To use it:
+
+Add `test-pilot-metrics` to your package.json file
+```javascript
+"dependencies": {
+  "test-pilot-metrics": "1.0.0",
+  ...
+}
+```
+
+Set up your Google Analytics credentials and enable your endpoints
+```javascript
+TODO: code here
+```
+
+Call testpilot.sendEvent()
+```javascript
+TODO: code here
+```
+
+The testpilot.sendEvent() function accepts these parameters:
```

Comment:
These fields are shared across the three systems with slightly different names so I think it will be easy to remap and send to all three as a base.  We'll still need to add specific fields for additional experiment meta-data.

This doesn't get us every measurement we could want, but it gets us a lot of data quickly